### DISCUSSION TITLE
Beeline guiding instructions

### DISCUSSION THREAD

**Discussion by 0709wiwiwi** [devemux86/cruiser#79] [Q&A] [answered]
Cruiser_Brouter_**beta** planner supports "routable" as well "beeline" navtracks.
In addition to turn commands, beeline clock guiding notations can be attached in a gpx navtrack.  
At recalculations, the beeline track sections are recognised and preserved (Cruiser_BRouter app). 
In navigation, offers the well known standard TTS turn commands this for the routable track paths.

**Comment by devemux86** [comment, marked as answer]
@0709wiwiwi thanks for the suggestion.

"Straight line" covers off-road areas, where Cruiser displays a direction arrow to the next waypoint.

**Comment by WolfgangZ42** [comment]
Although I very rarely participate in the DAKAR Ralley, I use the straight line function from time to time. Mostly when I got a .gpx file (from ACT or TET for instance) and OSM does not let me drive there. But then I have at least a path, which I can follow via overlay.

Here is a .kurviger file, the original .gpx (got from a MountainBiker) and a screenshot: 
[São_Bartolomeu_de_Messines.zip](https://github.com/devemux86/cruiser/files/10959865/Sao_Bartolomeu_de_Messines.zip)
![São_Bartolomeu_de_Messines](https://user-images.githubusercontent.com/121806238/224782525-de8e3d8f-f947-4ee8-ae36-47e7e605c9e9.png)

The guiding clock icon can certainly be interesting for desert sons. But a direction indicator to the next waypoint like it appears in a route lost situation: ![2021-05-08 14 55 41](https://user-images.githubusercontent.com/121806238/224777316-02ef69da-38f7-4d95-89cb-26756599fa8f.jpg) would be fine. Perhaps slightly modified:  ![2021-05-08 14 55 41](https://user-images.githubusercontent.com/121806238/224781692-4e558ccd-9fed-498f-bfcc-0820404c2685.jpg) to indicate straight line.

**Comment by abdullahO2** [comment]
As a person who lives in Saudi Arabia, which has vast desert areas and off-road tracks, I find this suggestion wonderful because off-road tracks are bumpy and the car vibrates in them, and this makes looking at the phone screen difficult, so this suggestion is very useful to know the direction without stopping and then Looking at the screen.

**Comment by devemux86** [comment]
Thanks everyone for your feedback!
I have already put it in my implementation list. :slightly_smiling_face:
